### ISSUE TITLE
Disable Classgraph Logging using JVM Argument?

### ISSUE THREAD

**Issue by landlockedsurfer** [classgraph/classgraph#438]
Hi all!
Not a bug report, but having a question here.
I'm getting ClassGraph log messages and would like to disable them. I assume that ClassGraph logging is enabled by one of my library dependencies. Is there a JVM argument to disable ClassGraph logging?
Or is there some way to configure the ClassGraph logger? As far as I understood ClassGraph uses its own logging mechanism to write standard error, correct? 

The output I'm getting looks like this:
```
...
12:04:19.678 [info] application - GET /login
12:04:20.398 [info] i.g.c.ClassGraph - 2020-06-16T12:04:19.933+0200      ClassGraph      ClassGraph version 4.8.53
2020-06-16T12:04:19.933+0200    ClassGraph      Operating system: Mac OS X 10.13.6 x86_64
2020-06-16T12:04:19.933+0200    ClassGraph      Java version: 1.8.0_242 / 1.8.0_242-b07 (Amazon.com Inc.)
...
```
Thanks!
Manfred

**Comment by lukehutch** [comment]
Hi Manfred,

In very old versions of ClassGraph, yes, ClassGraph just wrote log lines straight to stderr. However, for quite some time (and certainly well before the 4.8.53 version you see in the log), `java.util.logging.Logger.getLogger("io.github.classgraph.ClassGraph")` is used to get a `Logger` instance:

https://github.com/classgraph/classgraph/blob/master/src/main/java/nonapi/io/github/classgraph/utils/LogNode.java#L53

This is written to using `Logger.info`:

https://github.com/classgraph/classgraph/blob/master/src/main/java/nonapi/io/github/classgraph/utils/LogNode.java#L132

I'm not an expert in `Logger`, but one of these solutions might work for you:

https://stackoverflow.com/questions/2533227/how-can-i-disable-the-default-console-handler-while-using-the-java-logging-api

You can figure out which project has verbose logging enabled by calling `mvn dependency:tree`. You should report the issue to them, since they probably don't realize they shipped a version with verbose logging enabled.

**Comment by landlockedsurfer** [comment]
Thanks a lot! I was able to track down the cause of the surplus logging messages thanks to your advice. Since I'm using `sbt` instead of `mvn` I used the `sbt-dependency-graph` plugin to track down the transitive dependency `webjars-locator-core` required by `"org.webjars" %% "webjars-play" % "2.8.0"` as being the one triggering the `ClassGraph` logging.
The situation a little dodgy with my `logback` configuration, but I solved the problem now by requiring a later version of `webjars-locator-core`:
`"org.webjars" % "webjars-locator-core" % "0.45"`
Will also submited a pull request to the `webjars-play` project (basically bumping the required version).
